INSTALLATION.md

```diff
@@ -1,92 +1,91 @@
-how things are done or maybe participate in the improvement of the package,
-you can clone the GitHub repository:
+### 1 - Clone the repository
 
-```console
-$ git clone https://github.com/Avaiga/taipy-gui.git
+Clone the repository using the following command:
+```bash
+git clone https://github.com/Avaiga/taipy-gui.git
 ```
 
-This creates the 'taipy-gui' directory that holds all the package's source code.
-
-### Building the JavaScript bundle
+### 2 - Install Node.js
 
 Taipy GUI has some code dealing with the client side of the web applications.
-This code is written in [TypeScript](https://www.typescriptlang.org/), relies on
-[React](https://reactjs.org/) components, and is packaged into a JavaScript bundle
+This code is written in <a href="https://www.typescriptlang.org/" target="_blank">TypeScript</a>, relies on <a href="https://reactjs.org/" target="_blank">React</a> components, and is packaged into a JavaScript bundle
 that is sent to browsers when they connect to all Taipy GUI applications.
 
-Here is how you can build this bundle.
+This bundle needs to be built before being usable by Taipy GUI.
 
-You need to make sure that the [Node.js](https://nodejs.org/) JavaScript runtime version 18
-or above is installed on your machine.<br/>
-Note that Node.js comes with the [`npm` package manager](https://www.npmjs.com/) as part
-of the standard installation.
+First you need to install <a href="https://nodejs.org/" target="_blank">Node.js</a> on your system.
 
-Here is the sequence of commands that must be issued, assuming your current directory
-is the root directory of the repository:
+**Select the "Recommended For Most Users" version, and follow the instructions for your system.**
 
-```console
-# Current directory is the repository's root directory
-# Move to the 'gui' directory
-$ cd gui
-# Install the DOM dependencies (once and for all)
-$ cd dom
-$ npm i
-$ cd ..
-# Install the web app dependencies
-$ npm i --omit=optional
-# Build the web app
-$ npm run build
-$ cd ..
-# Current directory is the repository's root directory
+**Select "Automatically install the necessary tools" when asked.**
+
+### 3 - Build the JavaScript bundle
+
+Open a new terminal and **move to the gui directory** of the repository.
+
+Then, run the following commands:
+- Install the DOM dependencies
+```bash
+cd dom
+npm i
+cd ..
+```
+- Install the web app dependencies
+```bash
+npm i --omit=optional
+```
+- Build the web app
+```bash
+npm run build
 ```
 
-This creates the directory `src/taipy/gui/webapp` in the root directory of the repository
+After a few minutes, this creates the directory `src/taipy/gui/webapp` in the root directory of the repository
 where the front-end code for Taipy GUI is split into a set of JavaScript bundles.
 
-### Debugging the JavaScript bundle
+### 4 - Install the package as editable
+
+In a terminal, **at the root of the repository**, run:
+```bash	
+pip install -e . --user
```

Comment:
Is this how you work with dev version ?

Reply:
Yes, this is how I managed to install it. Is there something else I should add?

Reply:
no all good